typedb/cluster/session.py

```diff
@@ -23,8 +23,7 @@
 from typedb.api.options import TypeDBClusterOptions, TypeDBOptions
 from typedb.api.session import TypeDBSession, SessionType
 from typedb.api.transaction import TransactionType
-from typedb.cluster.database import _ClusterDatabase
-from typedb.cluster.failsafe_task import _FailsafeTask
+from typedb.cluster.database import _ClusterDatabase, _FailsafeTask
```

Comment:
Is this needed?

Reply:
It's needed because `typedb.cluster.failsafe_task` had to be deleted